scripts/check-no-machine-identifiers.test.sh

```
@@ -208,6 +216,381 @@ check "red: CSV header declares a hostname column" 1 \
+  "docs/note.md" 'Built at /Users/fixture-user/dev/SPOC on the laptop.'
+
+# The remedy the error message gives has to be the RIGHT remedy for the case
+# that produced this item: a switch path, whose fix is naming the switch.
+check "red: a switch directory says to name the switch" 1 \
+  "name the opam SWITCH, not its directory" \
+  "docs/note.md" 'opam exec --switch=/home/fixture-user/dev/SPOC -- dune build @fmt'
+
+# Portable spellings are OUT of scope, by decision, and that decision is pinned
+# rather than left to the regex: 78 tracked occurrences exist and all are fine.
+check "green: tilde and \$HOME spellings are not findings" 0 "no absolute home path" \
+  "docs/note.md" 'Install to ~/.local/bin, or "$HOME/.opam" / "${HOME}/.cache".'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)